CONTRIBUTING.md

```diff
@@ -1,229 +1,93 @@
 ```
 
-</td>
-</tr>
-<tr>
-<td>Cross-referencing internal files</td>
-<td>
-
-```rst
-
-:doc:`Title <../path/to/file/filename>`
-
-```
-
-</td>
-</tr>
-<tr>
-<td>External links</td>
-<td>
-
-```rst
-
-`link name  <URL>`_
-
-```
-
-</td>
-</tr>
-<tr>
-<tr>
-<td>Headings</td>
-<td>
-
-```rst
-
-******************
-Chapter title (H1)
-******************
-
-Section title (H2)
-===============
-
-Subsection title (H3)
----------------------
-
-Sub-subsection title (H4)
-^^^^^^^^^^^^^^^^^^^^
-
-
-```
-
-</td>
-</tr>
-<tr>
-<td>Images</td>
-<td>
-
-```rst
-
-.. image:: image1.png
-
-```
-
-</td>
-</tr>
-<tr>
-<td>Internal links</td>
-<td>
-
-```rst
-
-1. Add a tag to the section you want to reference:
-
-.. _my-section-tag: section-1
-
-Section 1
-==========
-
-2. Link to your tag:
-
-As shown in :ref:`section-1`.
-
-```
-
-</td>
-</tr>
-<tr>
-<tr>
-<td>Lists</td>
-<td>
-
-```rst
-
-# Ordered (numbered) list item
-
-* Unordered (bulleted) list item
-
-```
-
-</td>
-</tr>
-<tr>
-<tr>
-<td>Math (block)</td>
-<td>
-
-```rst
-
-.. math::
-
-  A = \begin{pmatrix}
-          0.0 & 1.0 & 1.0 & 3.0 \\
-          4.0 & 5.0 & 6.0 & 7.0 \\
-        \end{pmatrix}
-
-```
-
-</td>
-</tr>
-<tr>
-<td>Math (inline)</td>
-<td>
-
-```rst
-
-:math:`2 \times 2 `
-
-```
-
-</td>
-</tr>
-<tr>
-<td>Notes</td>
-<td>
-
-```rst
-
-.. note::
-
-  My note here.
-
-```
-
-</td>
-</tr>
-<tr>
-<td>Tables</td>
-<td>
-
-```rst
-
-.. csv-table::  Optional title here
-  :widths: 30, 70  #optional column widths
-  :header: "entry1 header", "entry2 header"
-
-   "entry1", "entry2"
-
-```
+### New feature development
 
-</td>
-</tr>
-</table>
+Use the [GitHub Discussion forum](https://github.com/RadeonOpenCompute/ROCm/discussions)
+(Ideas category) to propose new features. Our maintainers are happy to provide direction and
+feedback on feature development.
 
-## Language and style
+### Documentation
 
-We use the
-[Google developer documentation style guide](https://developers.google.com/style/highlights) to
-guide our content.
+Submit ROCm documentation changes to our
+[documentation repository](https://github.com/RadeonOpenCompute/ROCm). You must update
+documentation related to any new feature or API contribution.
 
-Font size and type, page layout, white space control, and other formatting
-details are controlled via
-[rocm-docs-core](https://github.com/RadeonOpenCompute/rocm-docs-core). If you want to notify us
-of any formatting issues, create a pull request in our
-[rocm-docs-core](https://github.com/RadeonOpenCompute/rocm-docs-core) GitHub repository.
+Note that each ROCm project uses its own repository for documentation.
 
-## Building our documentation
+## Future development workflow
 
-<!--  % TODO: Fix the link to be able to work at every files  -->
-To learn how to build our documentation, refer to
-[Building documentation](./building.md).
+The current ROCm development workflow is GitHub-based. If, in the future, we change this platform,
```

Comment:
we are github based for now and as far as we can see in the future. This section appears redundant.

Reply:
There was suggestion for the document the be GitHub agnostic.  Or at least highlight that we're currently using GitHub, but that can change, right, to cover bases.  I added these sections to address those concerns.

"I think there was general consensus around making any infrastructure requirements vendor agnostic in case we decide to switch at some point.  That said, the document could be construed as describing our current infrastructure process and some of the language could be made more generic and we could add a section explicitly stating that the current model uses GitHub, but that similar processes would be applicable if that infrastructure ever changed."